### PR TITLE
refactor(core): static-query migration should always use template strategy

### PR DESCRIPTION
**Pull request by devversion**
* See individual commits.

### CODE REVIEW

File: packages/core/schematics/migrations/static-queries/index.ts [hunk 1]
```diff
@@ -50,8 +55,9 @@ async function runMigration(tree: Tree, context: SchematicContext) {
   const logger = context.logger;
 
   logger.info('------ Static Query migration ------');
-  logger.info('In preparation for Ivy, developers can now explicitly specify the');
-  logger.info('timing of their queries. Read more about this here:');
+  logger.info('With Angular version 8, developers need to');
+  logger.info('explicitly specify the timing of ViewChild or');
+  logger.info('ContentChild queries. Read more about this here:');
   logger.info('https://github.com/angular/angular/pull/28810');
```
Comment (Splaktar) [inline packages/core/schematics/migrations/static-queries/index.ts]: When PR https://github.com/angular/angular/pull/30598 is merged and deployed to AIO, this URL should be:
https://angular.io/guide/deprecations#static-query-resolution

**Review by Splaktar — APPROVED** [review]
LGTM

File: packages/core/schematics/migrations/static-queries/index.ts (hunk 1 above)
Comment (kara) [reply packages/core/schematics/migrations/static-queries/index.ts]: Actually, we are going to link to a different guide (not the deprecation guide, so just leave this as is and I'll update it when the new guide is in.

**Review by kara — APPROVED** [review]
LGTM

**Comment by kara** [comment]
@alxhub did you want to review this for working with new TS signature?

**Comment by kara** [comment]
[presubmit](http://test/OCL:249574219:BASE:249574237:1558582806404:1e49eb0e)

**Comment by kara** [comment]
merge-assistance: code fresh

File: packages/core/schematics/migrations/static-queries/index.ts (hunk 1 above)
Comment (devversion) [reply packages/core/schematics/migrations/static-queries/index.ts]: Yes. At the time when I created that PR the official document for the static-query stuff is/was still WIP. We have an action-item to update the link but that is not related to this PR.

**Pull request closed without merging**